causely/README.md

```diff
@@ -0,0 +1,46 @@
+# Causely - Assure Continuous Application Reliability using Causal AI
+
+## Overview
+
+
+Our [Causal AI platform][1] automatically captures, represents and analyzes causality. We enable self-managed, resilient applications by bridging observability with automated orchestration.  Instead of constant noisy alerts, Causely tells you exactly what you need to know about all active and emerging application failures:
+
+- Why did it happen?
+- What was the root cause?
+- How can it be fixed?
+
+## What Causely offers:
+
+### Assure Continuous Application Reliability
```

Comment:
Within the overview, we avoid marketing material as this should be focused on: 
	1	What does your product do in 1-2 sentences
	2	What will customers get from this integration (the value of the integration & why it's helpful)
	3	What types of data sent and the value

Reply:
see below